lib/bolt/util.rb

```diff
@@ -77,6 +77,14 @@ def read_optional_yaml_hash(path, file_name)
         File.exist?(path) ? read_yaml_hash(path, file_name) : {}
       end
 
+      def first_runs_free
+        Bolt::Config.user_path + '.first_runs_free'
```

Comment:
Would using `analytics.yaml` instead of a new file work here? That'd prevent the welcome message displaying for existing users.

Reply:
We don't write `analytics.yaml` if the help text is printed, because then we have to load config which will significantly slow down this first help text run kind of unnecessarily. I don't think it's worth the tradeoff.